src/compiler/parser.ts

```diff
@@ -3015,7 +3015,31 @@ namespace ts {
                 let newPrecedence = getBinaryOperatorPrecedence();
 
                 // Check the precedence to see if we should "take" this operator
-                if (newPrecedence <= precedence) {
+                // - For left associative operator (all operator but **), only consume the operator

```

Comment:
You wrote "only consume the operator" here and below. I don't think there's anything special going on, so just write "consume the operator"